Changelog.MD

```diff
@@ -1,4 +1,11 @@
-Release notes:
+## Release notes:
+
+# 2.4.0 | 16 December 2020
+
+- Update icons to .png files to show in browsers again
+- Add link to options pages on firefox and chrome, to make it easier to switch feature on or off
+- Overhaul options page design.
+- Update options page to use vanilla js instead of jquery for the new design (it's 2020, we can use modern js now!)
```

Comment:
Sweet. Can we also work on removing jquery for the normal functioning of the extension. We already have a ticket for that i think #5